assets/src/components/vehicleIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+
+const VehicleIcon = ({
+  isSelected,
+  scale,
+}: {
+  isSelected?: boolean
+  scale?: number
+}) => {
+  const selectedClass = isSelected ? "selected" : ""
+  const scaleAmount = scale || 1.0
+
+  return (
+    <g className="m-vehicle-icon" transform={`scale(${scaleAmount})`}>
+      <path
+        className={`m-vehicle-icon__triangle ${selectedClass}`}
```

Comment:
The css only references the class `.m-vehicle-icon__triangle`, so the `.m-vehicle-icon` class could be removed.

Reply:
That’s true, but I still find it useful as a descriptive name.